easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.35.1-intel-2016a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'Gdk-Pixbuf'
+version = '2.35.1'
+
+homepage = 'https://developer.gnome.org/gdk-pixbuf/stable/'
+description = """
+ The Gdk Pixbuf is a toolkit for image loading and pixel buffer manipulation.
+ It is used by GTK+ 2 and GTK+ 3 to load and manipulate images. In the past it
+ was distributed as part of GTK+ 2 but it was split off into a separate package
+ in preparation for the change to GTK+ 3.
+"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+
+dependencies = [
+    ('GLib', '2.48.0'),
+    ('libjpeg-turbo', '1.4.2'),
```

Comment:
note: this was tweaked to use an alternative version that is compatible with NASM 2.12.01 in #2996